Lib/typing.py

```diff
@@ -662,9 +664,10 @@ class ForwardRef(_Final, _root=True):
 
     __slots__ = ('__forward_arg__', '__forward_code__',
                  '__forward_evaluated__', '__forward_value__',
-                 '__forward_is_argument__', '__forward_module__')
+                 '__forward_is_argument__', '__forward_is_class__',
+                 '__forward_module__')
 
-    def __init__(self, arg, is_argument=True, module=None):
+    def __init__(self, arg, is_argument=True, is_class=False, module=None):
```

Comment:
Since arguments to the initializer aren't kwarg-only, the only safe way to add it is at the end. Otherwise it's an API change and we can't accept it as a bugfix.
```suggestion
    def __init__(self, arg, is_argument=True, module=None, *, is_class=False):
```